src/sinol_make/contest_types/default.py

```diff
@@ -0,0 +1,17 @@
+from math import floor
+from typing import List
+
+from sinol_make.commands.run.structs import ExecutionResult
+
+
+class DefaultContest:
+    """
+    Default contest type.
+    """
```

Comment:
How can we make this comment more useful? Currently it almost copy-pastes the class name.

Reply:
I added some explanations